src/App.jsx

```diff
@@ -4,33 +4,125 @@ import {bestSellingTv, inventory} from "./constants/inventory.js";
+import ProductPrice2 from "./helpers/ProductPrijs.js";
 import Productsize from "./helpers/ProductFormaat.js";
+import Productsize2 from "./helpers/ProductFormaat.js";
 import minuss from './assets/minus.png';
 import checks from './assets/check.png';
+import './constants/oefening.js';
 
 
 function App() {
+
+    function sortgoedkoopste() {
+        inventory.sort((tv1, tv2) => {
+            return tv1.price - tv2.price;
+        });
+        console.log(inventory)
+    }
+
+    function sortmeesteverkocht() {
+        inventory.sort((tv1, tv2) => {
+            return tv2.sold - tv1.sold;
+        });
+        console.log(inventory)
+    }
+
+    function bestesporttv() {
+        inventory.sort((tv1, tv2) => {
+            return tv2.refreshRate - tv1.refreshRate;
+        });
+        console.log(inventory)
+    }
+
     return (
         <div>
             <header className="PageHead">
                 <h1>Tech it easy dashboard</h1>
             </header>
-            <main className="PageMain">
+            <main className={"PageMain"}>
                 <h2>Verkoop overzicht</h2>
-                <p1>Aantal producten {soldTvs(inventory)}</p1>
-                <p2>Aantal ingekochte producten {purchasedTvs(inventory)}</p2>
-                <p3>Aantal te verkopen producten {TvsToSell(inventory)}</p3>
+                <div className={"devider"}>
+                    <article className={"productdash1"}>
+                        <nav className={"nav1"}>
+                            <h3>Aantal verkochte producten</h3>
+                            <p1 className={"soltv"}>{soldTvs(inventory)}</p1>
+                        </nav>
+                    </article>
+                    <article className={"productdash2"}>
+                        <nav className={"nav2"}>
+                            <h3>Aantal ingekochte producten</h3>
+                            <p1>{purchasedTvs(inventory)}</p1>
+                        </nav>
+                    </article>
+                    <article className={"productdash3"}>
+                        <nav className={"nav3"}>
+                            <h3>Aantal te verkopen producten</h3>
+                            <p1> {TvsToSell(inventory)}</p1>
+                        </nav>
+                    </article>
+                </div>
+                <h2>Best verkochte tv</h2>
+                <div className={"devider2"}>
+                    <article className={"BestTV"}>
+                        <img src="https://image.coolblue.nl/max/500x500/products/1786196" alt="Tv"
+                             className={"Bestetv"}/>
+                        <br/>
+                        <div className={"gegevens"}>
+                            <p4>{ProductName(bestSellingTv)}</p4>
+                            <p5>{ProductPrice(bestSellingTv)}</p5>
+                            <p6>{Productsize(bestSellingTv)}</p6>
+                            <p7> wifi <img src={checks}/> speech <img src={minuss}/> hdr <img
```

Comment:
src verwacht hier een string. Als je die niet geeft krijgt je de volgende warning in webstorm: 'type {} is not assignable to type string'. Je kan dit verhelpen door alsnog een string eromheen te zetten met een template literal.